tests/settings.py

```diff
@@ -14,3 +14,10 @@
 # Set a fake api key for testing bug filing
 BZ_API_KEY = "12345helloworld"
 BZ_API_URL = "https://thisisnotbugzilla.org"
+
+# ELASTIC SEARCH
+# Configuration for elasticsearch backend
+ELASTIC_SEARCH.update({
+    "url": "127.0.0.1:9200",
```

Comment:
The URL for the Vagrant instance and Travis should be set via environment variables for both, and just the index prefix updated here.